integration/docker/entrypoint.sh

```diff
@@ -13,19 +13,20 @@
 set -e
 
 NO_FORMAT='--no-format'
+FUSE_OPTS='--fuse-opts'
 
 function printUsage {
   echo "Usage: COMMAND [COMMAND_OPTIONS]"
   echo
   echo "COMMAND is one of:"
-  echo -e " master [--no-format]         \t Start Alluxio master. If --no-format is specified, do not format"
-  echo -e " master-only [--no-format]    \t Start Alluxio master w/o job master. If --no-format is specified, do not format"
-  echo -e " worker [--no-format]         \t Start Alluxio worker. If --no-format is specified, do not format"
-  echo -e " worker-only [--no-format]    \t Start Alluxio worker w/o job worker. If --no-format is specified, do not format"
-  echo -e " job-master                   \t Start Alluxio job master"
-  echo -e " job-worker                   \t Start Alluxio job worker"
-  echo -e " proxy                        \t Start Alluxio proxy"
-  echo -e " fuse                         \t Start Alluxio fuse"
+  echo -e " master [--no-format]                                  \t Start Alluxio master. If --no-format is specified, do not format"
+  echo -e " master-only [--no-format]                             \t Start Alluxio master w/o job master. If --no-format is specified, do not format"
+  echo -e " worker [--no-format]                                  \t Start Alluxio worker. If --no-format is specified, do not format"
+  echo -e " worker-only [--no-format]                             \t Start Alluxio worker w/o job worker. If --no-format is specified, do not format"
+  echo -e " job-master                                            \t Start Alluxio job master"
+  echo -e " job-worker                                            \t Start Alluxio job worker"
+  echo -e " proxy                                                 \t Start Alluxio proxy"
+  echo -e " fuse [--fuse-opts=<fuse options separated by comma>]  \t Start Alluxio fuse with options for FUSE file systems"
```

Comment:
`fuse [--fuse-opts=opt1,...]  \t Start Alluxio FUSE file system, option --fuse-opts expects a list of fuse options separated by comma"`

Reply:
done